### PR TITLE
Refine make_att_files.py

### PR DESCRIPTION
**Pull request by alliecc**

### CODE REVIEW

File: argoverse/utils/make_att_files.py [hunk 1]
```diff
@@ -304,27 +306,22 @@ def compute_v_a(traj: np.array) -> Tuple[np.array, np.array]:  # traj:Nx3
...
-                        if dict_tracks[id_track]["length_track"] < 10 or dict_tracks[id_track]["exists"].sum() < 10:
-                            dict_tracks[id_track]["difficult_att"].append("short")
-                        else:
-                            if (ind_close_max - ind_close_min) - dict_tracks[id_track]["exists"][
-                                ind_close_min:ind_close_max
-                            ].sum() > 20:
-                                dict_tracks[id_track]["difficult_att"].append("occ")
+                        if (ind_close_max - ind_close_min) - dict_tracks[id_track]["exists"][
+                            ind_close_min:ind_close_max
+                        ].sum() > 20:
```
Comment (johnwlambert) [inline argoverse/utils/make_att_files.py:318]: can we make 20 a module-level private variable? e.g. `MAX_OCCLUSION_PCT_ = 20`

File: argoverse/utils/make_att_files.py [hunk 2]
```diff
@@ -304,27 +306,22 @@ def compute_v_a(traj: np.array) -> Tuple[np.array, np.array]:  # traj:Nx3
...
-                if dict_tracks[id_track]["length_track"] == 0:
-                    # this shouldn't happen
-                    dict_tracks[id_track]["difficult_att"].append("zero_length")
-
+                if dict_tracks[id_track]["list_dist"][ind_valid].min() > dist_close:
+                    dict_tracks[id_track]["difficult_att"].append("far")
                 else:
-                    if dict_tracks[id_track]["list_dist"][ind_valid].min() > dist_close:
-                        dict_tracks[id_track]["difficult_att"].append("far")
+                    if dict_tracks[id_track]["length_track"] < 10 or dict_tracks[id_track]["exists"].sum() < 10:
```
Comment (johnwlambert) [inline argoverse/utils/make_att_files.py:313]: can we make 10 a module-level private variable? e.g. `NEAR_DISTANCE_THRESH_ = 10`

File: argoverse/utils/make_att_files.py [hunk 3]
```diff
@@ -161,13 +163,13 @@ def compute_v_a(traj: np.array) -> Tuple[np.array, np.array]:  # traj:Nx3
         if np.sqrt((x_max - x_min) ** 2 + (y_max - y_min) ** 2) < 10:
```
Comment (johnwlambert) [inline argoverse/utils/make_att_files.py:163]: could we add a note about why a B-spline is needed in the function comment? I guess for occluded frames to get a constant tick rate (exactly 100 ms)?

File: argoverse/utils/make_att_files.py (hunk 2 above)
Comment (alliecc) [reply argoverse/utils/make_att_files.py:313]: It's not a distance threshold but a time threshold. I added a variable for it.

File: argoverse/utils/make_att_files.py (hunk 3 above)
Comment (alliecc) [reply argoverse/utils/make_att_files.py:163]: It's for general temporal smoothing. Not only for the occluded frames. I will add the comment.

File: argoverse/utils/make_att_files.py (hunk 1 above)
Comment (alliecc) [reply argoverse/utils/make_att_files.py:318]: ok

**Pull request merged**